rosdep/python.yaml

```diff
@@ -2912,9 +2912,11 @@ python-psycopg2:
   debian: [python-psycopg2]
   fedora: [python-psycopg2]
   gentoo: [=dev-python/psycopg-2*]
-  ubuntu:
-    '*': [python-psycopg2]
-    focal: [python3-psycopg2]
+  ubuntu: [python-psycopg2]
+python3-psycopg2:
```

Comment:
This is the right key, but you are going to have to move it down with the other `python3-` keys in order for it to pass the alphabetic linters.